Jenkinsfile

```diff
@@ -14,6 +14,12 @@ pipeline {
       }
     }
 
+    stage('Lint') {
```

Comment:
The `lint` step is covered by `npm run test`. So, no need this.

Reply:
It should be a separate stage.